packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -2450,6 +2450,14 @@
       },
       "datafeeds": {
         "type": "object"
+      },
+      "tags": {
+        "type": "text",
+        "fields": {
+          "keyword": {
+            "type": "keyword"
+          }
+        }
```

Comment:
Have you considered using the existing saved objects tagging functionality in `x-pack/plugins/saved_objects_tagging`? Using this would enable use of these tags in other parts of Kibana, like saved objects management.

Reply:
This is a good point, I had not considered it. However, giving it some thought now, I'm not sure they would be a good fit.
ML's modules have two versions, a JSON object which exists on disk (this is the vast majority of modules) and a saved object version which can be loaded by fleet. We currently only have 2 of these.
The saved object mappings should really be identical to the JSON version. Once they are loaded, they are treated the same.
Also the 2 fleet modules are not using these tags at the moment, but I've updated the mappings so they are in sync with the Module interface.